docs/hive-integration/README.md

```diff
@@ -162,10 +162,37 @@ In case that the Bare Metal Operator is installed, the Baremetal Agent Controlle
     - Hostname (optional for user to set)
     - MachineConfigPool (optional for user to set)
 - Reconcile the BareMetalHost hardware details by copying the Agent's inventory data to the BMH's `hardwaredetails` annotation.
+- Disable ironic inspection disabled
 
 
 See BMAC documentation [here](./baremetal-agent-controller.md).
 
+## Bare Metal Operator Integration using the converged flow
+See reference [here](https://github.com/openshift/enhancements/blob/master/enhancements/baremetal/ztp-metal3.md)
+
+In case the Bare Metal Operator on the HUB cluster version is 4.11 or higher, the integration between the MBO and the assisted-service is slightly different.
+The assisted-service will run a Preprovisioning Image Controller that reconcile the PreprovisioningImage for the  BareMetalHost.
+- Annotate the InfraEnv for the BareMetalHost with `infraenv.agent-install.openshift.io/enable-ironic-agent="true"`
+- Add the ironic agent config to the discovery ignition embeded to the InfraEnv ISO
+- Copy the InfraEnv ISODownloadURL to the PreprovisioningImage ImageUrl
+  This allows the host to register with metal3 and let metal3 reconcile the BareMetalHost hardware details.
+
+The Baremetal Agent Controller will will stop explicitly disabling *inspection* on `BareMetalHost`s it manages
+and will no longer update the BareMetalHost with detached annotation.
+It will continue to:
+- Find the right pairs of BMH/Agent using their MAC addresses
+- Reconcile the Agent's spec by copying the following attributes from the BMH's annotations:
+  - Role: master/worker
+  - Hostname (optional for user to set)
+  - MachineConfigPool (optional for user to set)
+- Set the  BareMetalHost CustomDeploy Method to `start_assisted_install` - this custom deploy method will be added to start
+  the assisted agent when BMO requests deployment.
+
+The assisted agent will not reboot the machine at the end of the installation, instead it will stop
+the assisted agent service and let the ironic agent to manage the machine power state
```

Comment:
just to verify - ai will write all the coreos-install flags before the reboot, right? It's only the reboot that is missing

Reply:
Yes

Reply:
Right.
Also, note that the converged flow preserves the host installer args overrides and ignition config overrides functionality (annotations on the BMH are copied to the agent)